### PR TITLE
Added display csc content for vacancy

### PR DESCRIPTION
**Pull request by gmackay**

### CODE REVIEW

File: vcm/src/main/java/uk/gov/cshr/vcm/model/Vacancy.java [hunk 1]
```diff
@@ -103,4 +103,7 @@
     @ManyToOne
     @JoinColumn(name = "dept_id")
     private Department department;
+
+    @Column(name = "displaycsccontent")
```
Comment (reachsaurabh) [inline vcm/src/main/java/uk/gov/cshr/vcm/model/Vacancy.java]: has the liquibase script been modified , if we merge this can cause SQL exception

Comment (reachsaurabh) [reply vcm/src/main/java/uk/gov/cshr/vcm/model/Vacancy.java]: I think the application will not start if validate is there already ,

Comment (gmackay) [reply vcm/src/main/java/uk/gov/cshr/vcm/model/Vacancy.java]: I updated the liquibase script first.  And swagger before that!

Comment (reachsaurabh) [reply vcm/src/main/java/uk/gov/cshr/vcm/model/Vacancy.java]: perfect

**Pull request merged**